### ISSUE TITLE
createImageBitmap breaks renderer.generateTexture

### ISSUE THREAD

**Issue by lukebro** [pixijs/pixijs#8219]
### Expected Behavior

`renderer.generateTexture` creates a texture from a Sprite.  This should work with both `CREATE_IMAGE_BITMAP` on and off.

### Current Behavior

Texture doesn't render when `CREATE_IMAGE_BITMAP` is on/true. The playground is pretty self explanatory: https://www.pixiplayground.com/#/edit/r_WuC81R7_PuLQx_gqFUs

### Possible Solution

No clue, but willing to explore solution if someone points me to the right direction or area. 

### Steps to Reproduce

Toggle on and off  the `CREATE_IMAGE_BITMAP` flag.

https://www.pixiplayground.com/#/edit/r_WuC81R7_PuLQx_gqFUs

### Environment

- **`pixi.js` version**: 6.2.0 (playground is 5.3.0 still broken)
- **Browser & Version**: Chrome 99.0.4844.51
- **OS & Version**: macOS Big Sir 11.6.1
- **Running Example**: https://www.pixiplayground.com/#/edit/r_WuC81R7_PuLQx_gqFUs

**Comment by ivanpopelyshev** [comment]
I think we need special flag for `generateTexture's` , in which case create_image_bitmap is not used inside.

**Comment by lukebro** [comment]
Yes I see what you mean, changing the `texture.once('update', onUpdate)` to a `texure.on` eventually callsback when the `createImageBitmap` resolves and `generateTexture` finally works.

https://www.pixiplayground.com/#/edit/g2tTEAjZXohqmWSBGxXXk

There isn't a good programatic way to respond to this though, the texture updates 3 times and all three times `texture.valid` is true.  Should `texture.valid` be false until the resource is fully loaded in this case?

**Comment by Zyie** [comment]
Hi there! 👋

We're doing some housekeeping and closing out older issues that haven't had recent activity.

PixiJS has evolved significantly, If this issue is still affecting you on PixiJS v8 (latest), please feel free to create a new issue with updated details and a minimal repro case.

Thanks for your understanding and for helping us keep the issue tracker focused and maintainable! 🩷
